src/test/scala/org/kibanaLoadTest/simulation/generic/GenericJourney.scala

```diff
@@ -211,6 +233,23 @@ class GenericJourney extends Simulation {
     s"test for ${journey.journeyName} ${config.version}"
   )
 
+  private val testData = journey.testData
+  if (!testData.isEmpty) {
```

Comment:
We should probably call `isDefined` instead? Taking a step back, if we only want to execute this code when the option is defined, we should probably map over the value.

Reply:
Scala [source](https://github.com/scala/scala/blob/2.13.x/src/library/scala/Option.scala#L169) says ` final def isDefined: Boolean = !isEmpty`, but I think isDefined is easier to read.